coins.json

```diff
@@ -4,6 +4,7 @@
         "symbol": "NAS",
         "decimals": 18,
         "blockchain": "Nebulas",
+        "slip44": 2718,
```

Comment:
Some pondering:
- Duplication.  It is good that this is a separate field now, it is easier to handle.  But duplication (present in slip44 and in derivation path) can be bad, can diverge, which can cause trouble.  OTOH, it allows purposefully different values (I have in mind subchains).
- Naming.  I know this is slip44 ID, but in most cases we just refer to it as coin type / coin ID.  I would call it `numID`, and in some special cases even deviate from slip44 (e.g. subchains).

Reply:
We have a string type id tho, I can rename slip44 to coinId, slip44 is not good variable

Reply:
Can we delete this value and read it from derivation path? It's not worth duplicating

Reply:
This might be confusing and will lead to bugs

Reply:
ok, let me revert it, maybe it's time to change to string based id? in some cases, slip44 id is not equal to derivation path

Reply:
In which coins slip44 not equal to deviation path? 

We can do slip:44

And then derivation path: `m/{{slip}}/0/`

As an option